### ISSUE TITLE
Sorry, I did not see the HItNet model implementent? Only StereoNet and AdaptiveNet are provided in code?

### ISSUE THREAD

**Issue by Magicboomliu** [meteorshowers/X-StereoLab#35]

**Comment by meteorshowers** [comment]
Yes, the current hitnet code performance of the pytorch version is 2.48 on [kitti.](http://www.cvlibs.net/datasets/kitti/eval_scene_flow_detail.php?benchmark=stereo&result=226494ba5559e9f5f46bdbd681d1564fee78409e). I am experimenting with the first author of the hitnet paper to check the potential problems of current version. The hitnet code will be released after all the work is done. This process may take several weeks.

**Comment by Magicboomliu** [comment]
ok, i got it. Thank you very much for your patience for open source the Hinet. I am looking forward to your upcoming version.

**Comment by meteorshowers** [comment]
@Magicboomliu   you can watch this code repository to get timely code updates.

**Comment by meteorshowers** [comment]
As my personal computing resources are relatively tight at the moment,  people  with adequate gpu computing resources are also welcome to contact me.

**Comment by meteorshowers** [comment]
For an experiment, generally 2 to 4 GPUs are needed. (2 is ok)

**Comment by xiaoxusanheyi** [comment]
model zoo 的模型文件请补充进去

**Comment by Magicboomliu** [comment]
您好，邮件已收到。感谢来信 ，我会尽快回复您。  Hello, I have already received the letter. Thanks for your mailing, I will reply as soon as possible.  こんにちは！確かご電子メールをもらいました。私はできるだけ早くあなたに返事を出します。

**Comment by xiaoxusanheyi** [comment]
感谢！祝你工作顺利




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "meteorshowers/X-StereoLab"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月12日(星期天) 中午1:23
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [meteorshowers/X-StereoLab] Sorry, I did not see the HItNet model implementent? Only StereoNet and AdaptiveNet are provided in code? (#35)





 &nbsp; 您好，邮件已收到。感谢来信 ，我会尽快回复您。&nbsp; Hello, I have already received the letter. Thanks for your mailing, I will reply as soon as possible.&nbsp; こんにちは！確かご電子メールをもらいました。私はできるだけ早くあなたに返事を出します。
